administration/index.md

```diff
@@ -6,13 +6,32 @@ layout: documentation
 
 # Overview
 
-In Work:
+As a base technology openHAB uses Karaf (i.e. for bundle management). [Apache Karaf](http://karaf.apache.org/) is an open source implementation of the [OSGi](https://en.wikipedia.org/wiki/OSGi) Release 5 core framework specification.
```

Comment:
This isn't really correct - Karaf is a container that runs an OSGi framework inside (which can be Equinox or Felix).